README.md

```diff
@@ -86,34 +119,39 @@ Make sure that you've verified that your build process uses that file as a sourc
 
 A short guide on verifying public keys and signatures should be available in the [docs](./docs/) directory.
 *******************************************************************************
-[your text here]
+Yes, These binaries created from the 15.8 release tar. Please refer to
+the ebuild for source: https://github.com/flatcar/scripts/blob/main/sdk_container/src/third_party/coreos-overlay/sys-boot/shim/shim-15.8-r2.ebuild#L9
```

Comment:
Could you add a second link to the Manifest next to the ebuild, that has the sha512 checksum

Reply:
@jepio The PR was merged after the last release, so we don't have shim-15.8-r2 in the release tags.

Reply:
Same here. Add tag to point to last release.